org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "0.6.3",
+    "sdk": "org.freedesktop.Sdk",
+    "separate-locales": "false",
```

Comment:
`separate-locales` is a boolean so it should be 
```json
 "separate-locales": false,
```

(no quotes around `false`)

flatpak-builder gives a warning about that.

Reply:
oh okay i'll update it